README.md

```diff
@@ -37,34 +37,34 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 <img src="docs/pandalion.png" alt="drawing" width="500"/>
 
-# DataFrame
-This is a C++ statistical library that provides an interface similar to Pandas package in Python.<BR>
+# Data Framing
```

Comment:
`Data Framing` doesn't make sense here. The name of the library is `DataFrame` and that's what I am describing